packages/mui-styled-engine-sc/package.json

```
@@ -50,7 +50,7 @@
     "@types/react": "^18.2.33",
     "chai": "^4.3.10",
     "react": "^18.2.0",
-    "styled-components": "^6.0.0"
+    "styled-components": "^6.1.0"
```

```suggestion
    "styled-components": "^6.0.0"
```

We should keep the minimal supported version in public packages.